common.mk

```diff
@@ -57,8 +59,11 @@ TOP_TARGETS = $(TOP_FILE) $(TOP_SMEMS_CONF) $(TOP_ANNO) $(TOP_FIR) $(sim_top_bla
 HARNESS_TARGETS = $(HARNESS_FILE) $(HARNESS_SMEMS_CONF) $(HARNESS_ANNO) $(HARNESS_FIR) $(sim_harness_blackboxes)
 
 # DOC include start: FirrtlCompiler
+# NOTE: These *_temp intermediate targets will get removed in favor of make 4.3 grouped targets (&: operator)
 .INTERMEDIATE: firrtl_temp
 $(TOP_TARGETS) $(HARNESS_TARGETS): firrtl_temp
+	touch $@
```

Comment:
I think it's confusing to use `.INTERMEDIATE` when there isn't a file being created (`firrtl_temp`). `.INTERMEDIATE` here is telling make that you'll generate a file called `firrtl_temp` that it can then delete. Are you sure this wasn't the problem? Let's discuss offline.

Reply:
Just to summarize the offline discussion: this `touch $@` solves the issue because any commands in this rule do so. We'll replace it with something that has fewer undesirable side effects (mangling timestamps is not great) like an `echo`.

Reply:
Offline result: We tested things a bit more. Specifically, we added a `touch firrtl_temp` equivalent and noticed that it didn't make a difference. However, any line added under `$(TOP_TARGETS) $(HARNESS_TARGETS): firrtl_temp` works (just needs a rule body). So I will change that to be something that doesn't affect the files (i.e. an echo or something similar. 

In general, this is a very confusing error to deal with. TL:DR `make` is busted wait for 4.3.